src/components/30-organisms/carousel/index.scss

```diff
@@ -0,0 +1,88 @@
+@mixin defaultAttributesChildElements {
+  @include typo(primary, large, default);
+  display: flex;
+  text-align: center;
+  justify-content: center; // align the dom element itself (f.e. small spans)
+}
+
+.o-carousel {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 5px; // for arrows
+  height: 100px; // set initial height because of IE bug with align-items and min-height
+  overflow: hidden;
+
+  :focus {
+    outline: none;
+  }
+
+  &__wrapper {
+    margin: 0 10px;
+
+    @include breakpoint($mediaquery-sm-up) {
+      margin: 0 30px;
+    }
+
+    ::slotted(*) {
+      @include defaultAttributesChildElements();
+    }
+  }
+
+  &__arrow {
+    padding: 6px;
+    height: 14px;
+    width: 6px;
+    border: solid $color-prim-white;
+    border-width: 2px 0 0 2px;
+    cursor: pointer;
+    background: transparent;
+
+    @include breakpoint($mediaquery-sm-up) {
+      padding: 10px;
+      height: 20px;
+      width: 10px;
+    }
+  }
+
+  &__arrow-left {
+    transform: rotate(-45deg);
+  }
+
+  &__arrow-right {
+    transform: rotate(135deg);
+  }
+
+  // animations
+  .animation-left {
+    animation: fromLeft 0.8s;
+  }
+
+  .animation-right {
+    animation: fromRight 0.8s;
+  }
+
+  @keyframes fromRight {
```

Comment:
very dangerous here as it pollutes global keyframes. Please use a more specific name like: `patternslibCarouselFromRight`